state/client.go

```diff
@@ -0,0 +1,50 @@
+package state
```

Comment:
Maybe move the content of `state.go` to this file? Other we'd have two separate files, both have methods for creating a new client

Reply:
Done in 9037389